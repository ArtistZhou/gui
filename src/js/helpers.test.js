import React from 'react';
import renderer from 'react-test-renderer';

// TODO: test ALL of the following!
import {
  fullyDecodeURI,
  isEncoded,
  statusToPercentage,
  decodeSessionToken,
  isEmpty,
  preformatWithRequestID,
  versionCompare,
  deepCompare,
  hashString,
  formatTime,
  formatPublicKey,
  intersection,
  customSort,
  mapDeviceAttributes,
  getFormattedSize,
  timeoutPromise,
  FileSize,
  collectAddressesFrom,
  getDemoDeviceAddress,
  detectOsIdentifier,
  findLocalIpAddress
} from './helpers';

describe('ExpandArtifact', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FileSize fileSize={1000} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
