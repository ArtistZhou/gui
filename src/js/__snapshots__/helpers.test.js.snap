// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandArtifact renders correctly 1`] = `
<div>
  1000.00 Bytes
</div>
`;
